src/instruction_handlers/common.rs

```diff
@@ -3,28 +3,67 @@ use crate::{addressing_modes::Arguments, instruction::ExecutionStatus, VirtualMa
 use eravm_stable_interface::{opcodes, OpcodeType, Tracer};
 
 #[inline(always)]
-pub(crate) fn instruction_boilerplate<Opcode: OpcodeType, T: Tracer, W>(
+pub(crate) fn boilerplate<Opcode: OpcodeType, T: Tracer, W>(
+    vm: &mut VirtualMachine<T, W>,
+    world: &mut W,
+    tracer: &mut T,
+    business_logic: impl FnOnce(&mut VirtualMachine<T, W>, &Arguments),
+) -> ExecutionStatus {
+    full_boilerplate::<Opcode, T, W>(vm, world, tracer, |vm, args, _, _| {
+        business_logic(vm, args);
+        ExecutionStatus::Running
+    })
+}
+
+#[inline(always)]
+pub(crate) fn boilerplate_w<Opcode: OpcodeType, T: Tracer, W>(
     vm: &mut VirtualMachine<T, W>,
     world: &mut W,
     tracer: &mut T,
     business_logic: impl FnOnce(&mut VirtualMachine<T, W>, &Arguments, &mut W),
 ) -> ExecutionStatus {
-    instruction_boilerplate_ext::<Opcode, T, W>(vm, world, tracer, |vm, args, _, world| {
+    full_boilerplate::<Opcode, T, W>(vm, world, tracer, |vm, args, world, _| {
         business_logic(vm, args, world);
         ExecutionStatus::Running
     })
 }
 
 #[inline(always)]
-pub(crate) fn instruction_boilerplate_ext<Opcode: OpcodeType, T: Tracer, W>(
+pub(crate) fn boilerplate_t<Opcode: OpcodeType, T: Tracer, W>(
```

Comment:
Bikeshedding / dumb question: Would using `boilerplate_wt` instead of `boilerplate_w` / `boilerplate_t` degrade performance? Subjectively, introducing 2 additional functions isn't really justified here; it's easy to use `_` for unused args, and it's not like there's a chance to mix up tracer and world args.

Reply:
Sounds reasonable.